### PR TITLE
lr changed to learning_rate in tf2.16.1 

### PR DESCRIPTION
**Pull request by lisatwyw**
[tf2.16.1 ](https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/Adam)
uses ```learning_rate``` rather than ```lr```

### CODE REVIEW

**Comment by egracheva** [comment]
Hi @lisatwyw ,

Thank you for your suggested changes to the learning rate naming. The first commit is relevant, and I'd like to accept and merge it. However, I noticed additional commits in this pull request that might have been added by mistake.

Could you please remove the extra commits so we can proceed with merging the relevant changes?
